bitnami/contour/values.yaml

```
@@ -559,10 +559,7 @@ envoy:
   ## @param envoy.updateStrategy [object] Strategy to use to update Pods
   ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   ##
-  updateStrategy:
-    type: RollingUpdate
-    rollingUpdate:
-      maxUnavailable: 10%
+  updateStrategy: {}
```

I agree that the current config can be problematic, but I wouldn't remove the `updateStrategy.type` param. I think we should follow the same approach we are using in other charts:
```suggestion
  ## @param envoy.updateStrategy [object] Strategy to use to update Pods
  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
  ## e.g:
  ## updateStrategy:
  ##  type: RollingUpdate
  ##  rollingUpdate:
  ##    maxSurge: 25%
  ##    maxUnavailable: 25%
  ##
  updateStrategy:
    type: RollingUpdate
```